.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: objective-c
+osx_image: xcode8.3
+
+env:
+  global:
+  - LC_CTYPE=en_US.UTF-8
+  - LANG=en_US.UTF-8
+  - PROJECT=OnDemandPassenger.xcodeproj
+  - SCHEME=OnDemandPassenger
+  - SDK=iphonesimulator10.3
+  - RUN_TESTS="NO"
+  - POD_LINT="NO"
+  - secure: "TWwwhkYnn9FyGJf+Bfg4VzAM0nI4nWKCDQV2pcB/6thlROQ4y8WNF4tobrxFHNwZryUvwaxXNbVGMEcAiLE709ybXXWdySY0dayoKopgQE9HYwGn0uvdmYVjzs4Fz+UZ/ml+oIxmSRnX1GAoA1bc+CSY/Pw7M5cTkSa0vjfVVuPtiKZAGmiYs2iYXWAl0GXrV67ZdNOG+0KBivwDWn3qrkMzdCLpK0frgQLBTnjrce/OGq0SyxFOvWmAO6m9uflVk4riPkSLU5pF9uVy3I8O1xsRWpnA+42ldKBKEeEE4hRmIn9lfg887RahcWNYSbfhy76seKw8H6nuYZPLXDKGscQaBuS08sQzS/ifYeEK7GiUiYplKDW8lwXxGAgAK+OPR8YdWuYJ/1N2mDmKvzkgQJERC7jXaQzwwO8QidxDo26JjcUagFgfEhw/6fV5OFc3Pj4oqRjpdvUHcKKEtbTqeNfTO4BuQsfGzciu+r+3xnSMp6CzJxF929L0c4fR8wMnm+MjFUJfBoVaZm7v/BFv47DqUJWpFkdM/FIJN7PFAktppmLmQMA5wfKzAiYC+ipwvobTbpeau0Xn1TMlho3uGz5LTk2wC1LpFiczQz5az2zeP+oaxxHKA2OBfJl6E1Pbjp9vu9saTbv9LJw/Anta9WaUPsPKdojJe/hEZ0HmPn8="
+  matrix:
+    - DESTINATION="OS=10.3,name=iPhone SE"
+    - DESTINATION="OS=9.0,name=iPhone 6"
+
+before_install:
+  - rm -rf /Users/travis/Library/Developer/Xcode/DerivedData/OnDemandPassenger-*/
+  - rvm use $RVM_RUBY_VERSION
+  - wget "$SDK_URL" -q -O HERE_iOS_SDK_Starter.tar.gz
+  - tar -xzf HERE_iOS_SDK_Starter.tar.gz
+  - cp -r HERE_iOS_SDK_Starter/framework/NMAKit.framework .
+
+script: 
+  - set -o pipefail
+  - xcodebuild -version
+  - xcodebuild -showsdks
+
+# Build App in Debug and Run Tests if specified
+  - if [ $RUN_TESTS == "YES" ]; then
+      travis_retry xcodebuild -project "$PROJECT" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO ENABLE_TESTABILITY=YES test;
+    else
+      travis_retry xcodebuild -project "$PROJECT" -sdk "$SDK" -destination "$DESTINATION" -configuration Debug ONLY_ACTIVE_ARCH=NO build;
+    fi
+ 
```

Comment:
I was just seeing GitHub's red icon saying "No newline at end of file", but the real issue is that while you have a newline after `fi`, you have trailing whitespace in line 36 (and line 25, BTW). Mind stripping these?